test/e2e/scheduling/preemption.go

```diff
@@ -186,10 +186,20 @@ var _ = SIGDescribe("SchedulerPreemption [Serial]", func() {
 				Limits:   podRes,
 			},
 		})
+		// Make sure that the lowest priority pod gets deleted
+		var podPreempted bool
+		// Wait till pod gets deleted
+		err := wait.PollImmediate(framework.Poll, framework.PodDeleteTimeout, func() (bool, error) {
+			preemptedPod, err := cs.CoreV1().Pods(pods[0].Namespace).Get(context.TODO(), pods[0].Name, metav1.GetOptions{})
+			if (err != nil && apierrors.IsNotFound(err)) || (err == nil && preemptedPod.DeletionTimestamp != nil) {
+				podPreempted = true
+				return podPreempted, nil
+			}
+			// Ignore transient errors
+			return false, err
+		})
```

Comment:
Can you pull this out into  a separate helper function? e.g. `podPreempted, err := waitForPodPreempted(pods[0].Namespace, pods[0].Name)`?

Reply:
Sure.